class.jetpack-network-sites-list-table.php

```diff
@@ -26,7 +26,10 @@ public function prepare_items() {
 		$this->process_bulk_action();
 
 		// Get sites
-		$sites = $jpms->wp_get_sites( array( 'exclude_blogs' => array( 1 ) ) );
+		$sites = $jpms->wp_get_sites( array(
+			'exclude_blogs' => array( 1 ),
+			'archived'		=> false,
```

Comment:
Tabs here can cause misaligned arrows, let's just use spaces, or not align at all?